File: Core/Models/Customer.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace Veloz.Core.Models
{
    public class Customer
    {
        public int Id { get; set; }
        [Required]
        [StringLength(100)]
        public string Name { get; set; }
        [StringLength(500)]
        public string Address { get; set; }
        [StringLength(500)]
        public string City { get; set; }
        [StringLength(50)]
        public string Phone { get; set; }
        [StringLength(50)]
        public string Email { get; set; }
        [StringLength(50)]
        public string SocialId { get; set; }
        [Required]
        public bool IsCorporate { get; set; } = false;
        [Required]
        public bool IsActive { get; set; } = true;
        public DateTime CreateDate { get; set; } = DateTime.Now;
        public DateTime LastUpdate { get; set; } = DateTime.Now;
    }
}

File: ClientApp/app/models/customer.ts
export interface Customer {
    id: number,
    name: string,
    address: string,
    city: string,
    phone: string,
    email: string,
    socialId: string,
    isCorporate: boolean,
    isActive: boolean,
    createDate: string,
    lastUpdate: string
}

export interface SaveCustomer {
    id: number,
    name: string,
    address: string,
    city: string,
    phone: string,
    email: string,
    socialId: string,
    isCorporate: boolean,
    isActive: boolean
}
